auto_backup/data/backup_data.yml

```diff
@@ -0,0 +1,28 @@
+# -*- coding: utf-8 -*-
```

Comment:
@pedrobaeza well if we still accept YML tests on 9.0 that's fine for me. The contrary would have annoyed me cause it would have slowed migration a bit too much... On the contrary, 9.0 to 10.0 migration should be pretty straightforward so it will be easier to bite the bullet.